### DISCUSSION TITLE
How to handle localization in our components

### DISCUSSION THREAD

**Discussion by caiotracera** [Gympass/yoga#626] [General]
Hey, people!
Here at Wellz we are working hard on making our end-user platform accessible. The major issue we are facing regarding Yoga is that some components have hard-coded words in English, but at the moment we only support Portuguese in our application. This causes a weird feeling with screen readers, mixing both idioms.

One example is the [`cleanable` option on the `Input` component](https://github.com/gympass/yoga/blob/master/packages/yoga/src/Input/web/Input.jsx#L161) (the screen reader reads `Clear`). The issue #550 is also related to this problem.

Given this introduction to the problem, what do you think could work to solve this?

**Comment by ericcleao** [comment]
Regarding the accessibility translations, I believe that any translation should be on the `Yoga` side once each project has its own way of handling translations and different languages.

We can probably discuss patterns to be used across all components like `naming` and which should be the props used on web and native, also how to define `roles`.

What do you think?

**Comment by rmsphd** [comment]
My suggestion is divided into two samples

1- The translation of internal parts
       - Like a Clear option of an input button
       - Need to solve as a theme in a global way,
example: https://mui.com/material-ui/guides/localization/
        - Yoga can provide the default only in English, and each client can set another English version or another language that the client wants. 

2- The translation of public parts
- Like a Label text, it needs to be translated using props on each component.
